pkg/signer/fuzz_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright The Rekor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signer
+
+import (
+	"os"
+	"testing"
+)
+
+func FuzzNewFile(f *testing.F) {
+	f.Fuzz(func(t *testing.T, keyData []byte, keyPass string) {
+		f, err := os.Create("keyfile")
```

Comment:
should this be `os.CreateTemp`?

Reply:
Or `t.TempDir()`? As that directory is automatically cleaned up.

Reply:
I.e create the file in the temporary directory :)

Reply:
> should this be `os.CreateTemp`?

Changed and rebased.